src/keybindings.c

```diff
@@ -824,6 +824,15 @@ static void apply_kb_accel(GeanyKeyGroup *group, GeanyKeyBinding *kb, gpointer u
 }
 
 
+/** Reloads keybinding settings from configuration file. Normally plugins do
```

Comment:
The first sentence should be separate, doxygen generates a brief one-liner for this.